.editorconfig

```diff
@@ -49,21 +50,311 @@ indent_size = 2
 # Sort using and Import directives with System.* appearing first
 dotnet_sort_system_directives_first = true
 # Avoid "this." and "Me." if not necessary
-#dotnet_style_qualification_for_field = false:suggestion
-#dotnet_style_qualification_for_property = false:suggestion
-#dotnet_style_qualification_for_method = false:suggestion
-#dotnet_style_qualification_for_event = false:suggestion
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
 
 csharp_style_inlined_variable_declaration = false:suggestion
 
 [*.cs]
```

Comment:
There are duplicated sections 2x`[*.cs]`, 2x`[*.{cs,vb}]`

We don't have vb code in our repo, so I guess we could combine all of the duplicated sections in one `[*.cs]`

Reply:
Yes, I copied most of the settings from the .NET squad.

Reply:
I found the issue!